host/testdata/es_os2_index_template.json

```diff
@@ -0,0 +1,116 @@
+{
+  "index_patterns": [
+    "test-visibility*"
+  ],
+  "template": {
+    "settings": {
+      "index": {
+        "number_of_shards": "5",
+        "number_of_replicas": "0"
+      }
+    },
+    "mappings": {
+      "dynamic": "false",
+      "properties": {
+        "DomainID": {
+          "type": "keyword"
+        },
+        "WorkflowID": {
+          "type": "keyword"
+        },
+        "RunID": {
+          "type": "keyword"
+        },
```

Comment:
pass this through `jq -S .` maybe?  that'll sort all keys, makes it a bit nicer if it's changed in the future.

Reply:
good idea, thanks